yarn.lock

```diff
@@ -2079,14 +2079,6 @@
   dependencies:
     nanoid "3.3.8"
 
-"@react-navigation/stack@^7.1.1":
```

Comment:
Where does this change come from if no libraries were updated? :thinking:

Reply:
Hmmm, this is weird

Reply:
Ah I think #263 bought these things in (while playing with the Stack Header), I forgot to remove another package too, will do that in this one itself